editor/src/clj/editor/game_object_non_editable.clj

```diff
@@ -163,9 +163,9 @@
     (distinct)
     (:components prototype-desc)))
 
-(defn prototype-desc->referenced-component-resources [prototype-desc workspace]
+(defn prototype-desc->referenced-component-resources [prototype-desc proj-path->resource]
   (eduction
-    (map (partial workspace/resolve-workspace-resource workspace))
+    (map proj-path->resource)
```

Comment:
Unrelated fix. I discovered we had not fully transitioned to in-transaction `proj-path` resolution introduced in #10208 here.